stubs/solvers.py

```diff
@@ -253,7 +252,7 @@ def init_petsc_matnest(self):
                     Jpetsc.append(d.PETScMatrix(Jsum))
 
         for j in Jpetsc:
-            print(type(j))
+            logger.debug(type(j))
```

Comment:
This for loop can be removed

Reply:
Fixed in [f8eaa1a](https://github.com/RangamaniLabUCSD/stubs/pull/82/commits/f8eaa1aaec9d83148ef6dac5741a4913832d799b)